crowdfunding_compassion/views/project_donation_form_page.xml

```diff
@@ -0,0 +1,154 @@
+                    <div class="container">
+                        <h1 class="hero-title">Make your donation</h1>
+
+                        <div class="row mt-4">
+                            <!-- Amount selector -->
+                            <div class="col-sm-12 col-md-8">
+                                <h4 class="white">
+                                    I want to help
+                                    <t t-esc="project.name"/>
+                                    funding for
+                                </h4>
+                                <span class="white">(Click one option to select)</span>
+
+                                <div class="row mt-3">
+
+                                    <!-- Three example quantities are displayed, set the 3 default number of product for each here -->
+                                    <t t-set="fund_default_quantities" t-value="[1, 3, 5]" />
+                                    <t t-foreach="fund_default_quantities" t-as="current_default_quantity">
+                                        <t t-call="crowdfunding_compassion.fund_amount_example" />
+                                    </t>
+
+                                    <!-- Custom amount -->
+                                    <div class="col-6 col-lg-3 mb-3">
+                                        <label class="h-100 w-100">
+                                            <input type="radio" name="amount" value="" class="card-input-element d-none" id="custom-amount-card"/>
+                                            <div class="card h-100 d-flex justify-content-around align-items-center text-center">
+                                                <h3 class="blue">Another amount</h3>
+                                                <input type="number" id="custom-amount-field"/>
+                                                <h3 class="blue">CHF</h3>
+                                            </div>
+                                        </label>
+                                    </div>
+                                </div>
+                            </div>
+
+                            <!-- Project card -->
+                            <div class="col-sm-12 col-md-4">
+                                <t t-set="display_impact" t-value="True" />
+                                <t t-call="crowdfunding_compassion.project_impact_card" />
+                            </div>
+                        </div>
+                    </div>
+                </section>
+
+                <section>
+                    <div class="container mb-5">
+                        <h3 class="blue uppercase">Your billing information</h3>
+                        <div t-attf-class="{{form.form_wrapper_css_klass}}">
+                            <t t-raw="form.form_render()"/>
+                        </div>
+                    </div>
+                </section>
+            </t>
+
+            <script>
```

Comment:
Shouldn't the script be placed in a separate js file and loaded properly with assets_frontend? You could also make benefit from jQuery library.

Reply:
Sure, I will look into it along with my next task so that we can merge this PR.